packages/network/src/api/image/requests.ts

```diff
@@ -323,6 +332,12 @@ function createLocalImage(options: AddImageOptions): Image {
   return image;
 }
 
+function precacheThumbnail(options: AddImageOptions, image: Image): Promise<void> {
+  return options.uploadType !== ImageUploadType.VIDEO_FRAME && options.useThumbnailCaching !== false
+    ? ky.get(image.thumbnailPath).then(() => {})
+    : Promise.resolve();
```

Comment:
I think the `if` statement to check if the request should be made should not be done in this function. This function is called `precacheThumbnail` so it should just do what it is supposed to do : make a precache request.

The conditions to check if the request should be made or not should be in the add image request.